packages/kit/src/runtime/client/client.js

```diff
@@ -1236,10 +1249,14 @@ export function create_client({ target, base }) {
 			observer.disconnect();
 
 			for (const a of target.querySelectorAll('a')) {
-				const { url, external, options } = find_anchor(a, base);
-
+				const { url, external } = get_link_info(a, base);
 				if (external) continue;
 
+				const options = get_router_options(a);
+				if (options.reload) continue;
+
+				if (external || options.reload) continue;
```

Comment:
```suggestion
```

This is a remnant of my previous suggestion. Github didn't let me include this line in the selection because there was a deleted line in between.

Reply:
Thanks, we also need to delete some of the empty line or lint would fail, removed in a separate commit.